src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -97,7 +97,7 @@ public void ChatTestsStaff_003_sendPrivateMessage() {
     chatUserOne.verifyMessageOnChat(userTwoPrivateMessage);
   }
 
-  @DontRun(env = {"prod"})
+  @DontRun(env = {"prod", "preview"})
   @Test(groups = {"ChatTestsStaff_004", "ChatStaff", "ChatTests"})
   public void ChatTestsStaff_004_basicUserChatFails() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)